Makefile

```diff
@@ -41,6 +41,16 @@ apply-crd:
 	@kubectl delete crd loadtests.kangal.hellofresh.com || true
 	@kubectl apply -f charts/kangal/crd.yaml
 
+# Install required protoc plugins
+protoc-plugins:
+	@printf "$(OK_COLOR)==> Installing required protoc plugins$(NO_COLOR)\n"
+	@go install \
+	   github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
+	   github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
+	   google.golang.org/protobuf/cmd/protoc-gen-go \
+	   google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

Comment:
Do we really need to install things system wide? Can't we use something like the following to also ensure the correct version.
```
	BIN_DIR=...
	BUILD_DIR=$$(mktemp -d); \
	cd "$${BUILD_DIR}"; \
	go mod init protoc-plugins; \
	go get "github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@v2.0.1"; \
	go build -mod=readonly -o "$${BIN_DIR}/protoc-gen-grpc-gateway" github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway; \
	rm -rf "$${BUILD_DIR}"
	"$${BIN_DIR}/protoc-gen-grpc-gateway" --version
```

Reply:
These binaries are `protoc` plugins, installing them like this makes the whole set up much more complicated. And we expect `protoc` installed globally as well.

Reply:
Should we have protoc installed globally? Are we going to support protoc v1, v2, v3 etc?
By not providing versions we actually get a build system that will very every single time something is build.

Reply:
We are using latest stable `protoc` version that is `3.x` - it is pinned in the CI and in the dev instruction.